index.html

```diff
@@ -161,32 +169,38 @@ <h2 class="title cards">Multi-Post Stories</h2>
             </li>
         </ul>
 
-        <a href="#" class="button project">See Project</a>
+        <a href="#" class="button project" id="last">See Project</a>
     </div>
+</div>
     </section>
 
     <section id="myself">
 
-        <h2 class="title">
+        <div>
+            <h2 class="title" id="last_h2">
                 About Myself
             </h2>
-            <p class="text">
+            <p class="text" id="last_p">
                 I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
             </p>
-            <p id="color">LET’S CONNECT</p>
+            <p id="color" class="last">LET’S CONNECT</p>
 
-            <ul class="social" id="social">
+            <ul class="social end" id="social">
 
                 <li><a href="#"><img src="images/git.svg" id="github" class="icon"></a></li> 
                 <li><a href="#"><img src="images/hand.svg" id="hand" class="icon"></a></li>
                <li><a href="#"><img src="images/linkedin.svg" id="linkedin" class="icon"></a></li>
                <li><a href="#"><img src="images/twitter.svg" id="twitter" class="icon"></a></li>
                 <li><a href="#"><img src="images/M.svg" id="m" class="icon"></a></li>
-            </ul>
-
+            </ul> 
             <p id="resume">Get my resume</p>
+        </div>
+
+        
+
+           
 
-            <img src="images/skills.png" alt="skills_list">
+            <img src="images/skills.png" alt="skills_list" id="skills">
```

Comment:
- [ ] Kindly fix the following.

- Your design
<img width="434" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/fad2b7d4-ef8f-4d65-a7fc-df65cf2d6369">

- Figma design
<img width="413" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/0ffe8918-89ec-43c8-a3c1-3e9c8420ce91">

- 1. Increase the font size, reduce the margin of wherever has arrows, and increase the left and right padding of the text under the `Contact me` heading to match the design.
- 2. Fix the padding of the button.

Reply:
As suggested by the previous reviewer, kindly fix the padding of the button, it looks high on the left and right side of the text